surveys/2022-wg-debugging-survey/questions.md

```diff
@@ -0,0 +1,361 @@
+# Survey questions
+One of the [compiler team's focuses for
+2022](https://blog.rust-lang.org/inside-rust/2022/02/22/compiler-team-ambitions-2022.html)
+is to improve the experience of debugging Rust code. We'd like to better
+understand your experiences using a debugger with Rust - what's been good,
+what's been bad and how your experience debugging Rust could be better.
+
+Unless you choose to enter your email, your answers will be anonymous. Any
```

Comment:
Are you actually collecting an email address? If not this line should be changed

Reply:
Fixed - not collecting an email